objectivec/src/ort_enums.mm

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "ort_enums_internal.h"
+
+#include <algorithm>
+
+#include "core/session/onnxruntime_cxx_api.h"
+
+namespace {
+struct LoggingLevelInfo {
+  ORTLoggingLevel logging_level;
+  OrtLoggingLevel capi_logging_level;
+};
+
+// supported ORT logging levels
+// define the mapping from ORTLoggingLevel to C API OrtLoggingLevel here
+constexpr LoggingLevelInfo kLoggingLevelInfos[]{
+    {ORTLoggingLevelVerbose, ORT_LOGGING_LEVEL_VERBOSE},
+    {ORTLoggingLevelInfo, ORT_LOGGING_LEVEL_INFO},
+    {ORTLoggingLevelWarning, ORT_LOGGING_LEVEL_WARNING},
+    {ORTLoggingLevelError, ORT_LOGGING_LEVEL_ERROR},
+    {ORTLoggingLevelFatal, ORT_LOGGING_LEVEL_FATAL},
+};
+
+struct ValueTypeInfo {
+  ORTValueType type;
+  ONNXType capi_type;
+};
+
+// supported ORT value types
+// define the mapping from ORTValueType to C API ONNXType here
+constexpr ValueTypeInfo kValueTypeInfos[]{
+    {ORTValueTypeUnknown, ONNX_TYPE_UNKNOWN},
+    {ORTValueTypeTensor, ONNX_TYPE_TENSOR},
+};
+
+struct TensorElementTypeInfo {
+  ORTTensorElementDataType type;
+  ONNXTensorElementDataType capi_type;
+  size_t element_size;
+};
+
+// supported ORT tensor element data types
+// define the mapping from ORTTensorElementDataType to C API
+// ONNXTensorElementDataType here
+constexpr TensorElementTypeInfo kElementTypeInfos[]{
+    {ORTTensorElementDataTypeUndefined, ONNX_TENSOR_ELEMENT_DATA_TYPE_UNDEFINED, 0},
+    {ORTTensorElementDataTypeFloat, ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT, sizeof(float)},
+    {ORTTensorElementDataTypeInt32, ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32, sizeof(int32_t)},
+};
```

Comment:
Is it possible to expand this list in the near future (e.g. support 8-bit input for quantized models)?

Reply:
yes, it is